randovania/game_connection/connector/debug_remote_connector.py

```diff
@@ -22,15 +21,16 @@ class DebugRemoteConnector(RemoteConnector):
     _finished: bool = False
     _last_inventory_event: Inventory
     item_collection: ResourceCollection
-
-    RemotePickupsUpdated = Signal()
-    MessagesUpdated = Signal()
+    RemotePickupsUpdated: RdvSignal
```

Comment:
shouldn't this be `RdvSignal[]` or is that not a thing?

Reply:
No, because then it expects `index or slice expression`

Reply:
strictly speaking it has to be `RdvSignal[()]`. if we upgrade to py3.13 we could use typevar defaults to make just `RdvSignal` work

Reply:
Though after some troubleshooting, it probably should be `RdvSignal[()]` to pass mypy.

Reply:
Alternatively, the argument to RdvSignal could be arguments to Callable. So we'd do `RdvSignal[[int, str], None]`

Goes along with having return values of the listeners being the result of `emit`, but that's very not much not needed